### DISCUSSION TITLE
[Feature] Remove exact duplicates

### DISCUSSION THREAD

**Discussion by hermesespinola** [immich-app/immich#10386] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Now that Immich supports finding Duplicate fotos and manually reviewing them before deleting, it would be useful for me to automatically remove exact duplicates (I messed up some external library import somehow and I have thousands of duplicate photos). I wonder if this would be useful for more people. How easy would it be to implement this given that (I assume) photos are already tagged with some kind of similarity score?

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
If you mean exact duplicates on a byte level - Immich already rejects those for the upload library. For external libraries it doesn't in order to most precisely represent what's on disk (since that's presumably what people want).
